src/sasl/cyrsasl_scram.erl

```diff
@@ -187,27 +211,36 @@ create_server_first_message(ClientNonce, ServerNonce, Salt, IterationCount) ->
         jlib:encode_base64(Salt), <<",i=">>, integer_to_list(IterationCount)]).
 
 -spec parse_step4_client_in(client_in()) ->
```

Comment:
I think this function is not taking a `client_in()` alone anymore 🤔

Reply:
good catch